service-worker/index.js

```diff
@@ -99,12 +133,14 @@ const CUSTOM_FETCH = (event) => {
 
 // Force put to sw-cache
 const CUSTOM_PUT = (event) => {
-  console.log(`SW:: Custom sw-cache-put has been triggered for '${event.data.url}'`);
+  console.log(`SW::CUSTOM_PUT:: Custom sw-cache-put has been triggered for '${event.data.url}'`);
 
   let request = new Request(event.data.url, event.data.options);
   
+  let modifiedHeaders = new Headers([...event.data.options.headers, ['from-sw', true]]);
```

Comment:
Also, let's keep the setting of `from-sw: true` in the response headers in a single place via single method.

Reply:
This method has a different pattern for response build, hence unable to move it to the common function(CACHE_PUT_TO_SW).